dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowFactoryImpl.java

```diff
@@ -775,23 +792,27 @@ public boolean existSchemeIdOnSchemesList(String schemeId, List<WorkflowScheme>
 	@Override
 	public WorkflowTask findTaskByContentlet(final Contentlet contentlet) throws DotDataException {
 		if (!UtilMethods.isSet(contentlet.getIdentifier()) || cache.is404(contentlet)) {
-			return new WorkflowTask();
+			return null;

```

Comment:
I would prefer to return a workflowtask which is initially init on null, if the validation pass, do the process otherwise return the workflowtask with null.

Reply:
@jdotcms  we discuss this last week. In the meeting was decided to use null instead of the Workflow Task  init with nulls.